app/api/auth.py

```diff
@@ -39,12 +39,20 @@ def basic_auth_error():
 #     return user.check_password(password)
 
 
+# @token_auth.verify_token
+# def verify_token(token):
+#     g.current_user = User.check_token(token) if token else None
+#     print(g.current_user)
+#     return g.current_user is not None
+
 @token_auth.verify_token
 def verify_token(token):
-    g.current_user = User.check_token(token) if token else None
-    print(g.current_user)
-    return g.current_user is not None
-
+   print("why")
+   g.current_user = Sitter.check_token(token) if token else None
+   if not g.current_user:
+       g.current_user = Owner.check_token(token) if token else None
```

Comment:
I get what you are doing here, I think it's fine, but something cleaner in the future is to have one function for checking token, and to store in the token the type of object (when you get_token)

Reply:
it seems @wangtianxing1010  and @Rhut-virani,  we both have to talk to you about the decorator and how to include the things what you are suggesting into the decorator.